modules/nw-pod-network-connectivity-configuration.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/verifying-connectivity-endpoint.adoc
+
+[id="nw-pod-network-connectivity-configuration_{context}"]
+= Configuring pod connectivity check placement
+
+As a cluster administrator, you can configure on which nodes the connectivity check pods run by modifying the `network.config.openshift.io` object named `cluster`.
+
+.Prerequisites
+
+* Install the {oc-first}.
+
+.Procedure
+
+. To edit the connectivity check configuration, enter the following command:
++
+[source,terminal]
+----
+$ oc edit network.config.openshift.io cluster
+----
+
+. In the text editor, update the `networkDiagnostics` stanza to specify the node selectors that you want for the source and target pods.
+
+. To commit your changes, save your changes and exit the text editor.
+
+.Verification
+
+To verify that the source and target pods are running on the intended nodes, enter the following command:
```

Comment:
I think it is also worth mentioning the `NetworkDiagnosticsAvailable` condition in `cluster/network.config.openshift.io/v1`, for example:
```
  status:
...
    conditions:
    - lastTransitionTime: "2024-05-27T08:28:39Z"
      message: ""
      reason: AsExpected
      status: "True"
      type: NetworkDiagnosticsAvailable
```